src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const rules = sourceString.split(';');
+
+  rules.forEach((rule) => {
+    const clearRule = rule.trim();
+
+    if (clearRule) {
+      const parts = clearRule.split(':');
+      const key = parts[0].trim();
+      const value = parts.length > 1 ? parts.slice(1).join(':').trim() : '';
```

Comment:
The current implementation assumes that the value is everything after the first colon. This could lead to unexpected behavior if the value itself contains colons. Consider whether this behavior is intended and if not, adjust the logic to handle such cases appropriately.